src/System.Runtime/tests/System.Runtime.Tests.targets

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <Target Name="CopyFilesToNonTPADir" BeforeTargets="RunTestsForProject" >
```

Comment:
where would this SupplementalTestData files be placed. I don't want this file to placed in test root directory as it gets added to TPA list.